plugin.video.aod/resources/language/English/strings.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
could you convert the xmls to .po files, please? Language xml files are deprecated.